event-management-frontend/src/components/eventCreateOrEdit/locationSection/MapDisplaySearchMarker.tsx

```diff
@@ -19,32 +19,41 @@ const MapDisplaySearchMarker = (props: Props) => {
   const classes = useStyles();
   const { t } = useTranslation();
 
+  const [show, setShow] = useState(true);
+  const closePopup = () => {
+    setShow(false);
+  };
+  useEffect(() => setShow(true));
```

Comment:
This instruction sets `show` to true after the UI is rendered but you already have it on true because of the initial value you passed the state. Do you want to achieve something else with this?